OpenRA.Mods.Common/UtilityCommands/UpgradeMapCommand.cs

```diff
@@ -65,13 +75,29 @@ public static void UpgradeMap(ModData modData, IReadWritePackage package, int en
 				return;
 			}
 
-			var map = new Map(modData, package);
-			ProcessYaml(modData, map, map.WeaponDefinitions, engineDate, UpgradeRules.UpgradeWeaponRules);
-			ProcessYaml(modData, map, map.RuleDefinitions, engineDate, UpgradeRules.UpgradeActorRules);
-			ProcessYaml(modData, map, map.SequenceDefinitions, engineDate, UpgradeRules.UpgradeSequences);
-			UpgradeRules.UpgradePlayers(modData, engineDate, ref map.PlayerDefinitions, null, 0);
-			UpgradeRules.UpgradeActors(modData, engineDate, ref map.ActorDefinitions, null, 0);
-			map.Save(package);
+			var yaml = new MiniYaml(null, MiniYaml.FromStream(package.GetStream("map.yaml"), package.Name));
```

Comment:
This will crash with an NRE later on if there is no `map.yaml` file present in the directory. Can we get a proper error message here instead? (Or ignore such maps/folders.)